docs/user/build/install/jdk.rst

```diff
@@ -252,18 +252,18 @@ This is only needed if the windows one-click installers don't work for you:
 * Perhaps you are on Linux?
 * Perhaps you are on Mac and the version of JAI/ImageIO included with your operating system is out of date?
 
-The goal is to place the required jars into your lib/ext directory of both your JDK (for compiling) and your JRE (for running).
+The goal is to place the required jars into your ``lib/ext`` directory of both your JDK (for compiling) and your JRE (for running).
 
-Optional: macOS ImageIO
-'''''''''''''''''''''''
+Optional: mac OS ImageIO
```

Comment:
Wikipedia suggests macOS - see https://en.wikipedia.org/wiki/MacOS#macOS

Also a couple of lines down.

Probably just add that to the local dictionary?